maps/FastShipCabins_SE_SSE_CaptainsCabin.asm

```diff
@@ -18,34 +18,34 @@ FastShipCabins_SE_SSE_CaptainsCabin_MapScriptHeader:
 .MapCallbacks:
 	db 0
 
-UnknownScript_0x75ea6:
+Unused_0x75ea6:
 	end
 
-CaptainScript_0x75ea7:
+Captain:
```

Comment:
It would be hard to know what this label was referring to at a glance. I think `SSAquaCaptainScript` would be more consistent with the other labels in here like `SSAquaCaptainExhaustingText`.

Reply:
Good point!  That's fixed now.  He's the SSAquaCaptain.